lms/static/sass/discussion/elements/_editor.scss

```diff
@@ -22,6 +22,11 @@
 
 .wmd-preview-container {
   @include discussion-new-post-wmd-preview-container;
+
+  .discussion-board &,
```

Comment:
It is quite a hack here. Because the gap is noticeable in a UXPL page. The reason to put it here is when WMD is replaced, it no longer affects anything else, and no effort to clean up in Board and Profile page scss. Pretty neat.